sidecar/internal/service/reclaimspace.go

```diff
@@ -61,13 +69,35 @@ func (rs *ReclaimSpaceServer) ControllerReclaimSpace(
 	pvName := req.GetPvName()
 	klog.Info(pvName)
 
-	// TODO: get following params using kubeClient and pvName
+	pv, err := rs.kubeClient.CoreV1().PersistentVolumes().Get(ctx, pvName, metav1.GetOptions{})
+	if err != nil {
+		klog.Errorf("Failed to get pv: %v", err)
+		return nil, status.Errorf(codes.InvalidArgument, "failed to get pv")
```

Comment:
include the name of the pv in the error, like `status.Errorf(codes.InvalidArgument, "failed to get pv %q", pvName)`. When a failure is returned and logged, it helps the person troubleshooting it a lot to have these details.